domain/learningandworkprogress/src/main/kotlin/uk/gov/justice/digital/hmpps/domain/learningandworkprogress/review/Exceptions.kt

```diff
@@ -14,3 +14,12 @@ class ActiveReviewScheduleAlreadyExistsException(val prisonNumber: String) :
 
 class InvalidReviewScheduleStatusException(val prisonNumber: String, fromStatus: ReviewScheduleStatus, toStatus: ReviewScheduleStatus) :
   RuntimeException("Invalid Review Schedule status transition for prisoner [$prisonNumber] status from $fromStatus to $toStatus")
+
+sealed class InvalidReviewScheduleException(val prisonNumber: String, val sentenceType: SentenceType, message: String) : RuntimeException(message) {
```

Comment:
Specific exemption types for when the review schedule would be invalid

This is to replace the UnsuportedOperationException that we were originally throwing in the service code